store/json.go

```diff
@@ -176,16 +176,20 @@ func (kvs *jsonFileStore) flush() error {
 
 // Lock locks the store for exclusive access.
 func (kvs *jsonFileStore) Lock(block bool) error {
+	var (
+		lockFile *os.File
+		err      error
+	)
+
 	kvs.Mutex.Lock()
 	defer kvs.Mutex.Unlock()
 
 	if kvs.locked {
 		return ErrStoreLocked
 	}
 
-	var lockFile *os.File
-	var err error
-	lockPerm := os.FileMode(0o664) + os.FileMode(os.ModeExclusive)
```

Comment:
was this value `0o664` valid?

Reply:
i changed to 0664 which is valid.

Reply:
@jsturtevant `0o644` is a legal (and unambiguous) representation of 0644, where the little `o` indicates octal

Reply:
@tamilmani1989 you uncorrected it, 0o644 is less ambiguous and therefore preferred

Reply:
@rbtr oops. thanks for letting me know. i will revert this back

Reply:
@rbtr thanks for the explanation!